docs/docs/responses.mdx

```diff
@@ -258,3 +258,31 @@ responses:
   - text: "Hey, {name}. How are you?"
   - text: "Hey, {name}. How is your day going?"
 ```
+
+## Responses for Retrieval Intents
+
+If you are using retrieval intents in your assistant, you also need to add response templates
+for your assistant's replies to these intents in a separate file:
```

Comment:
I thought response selector intents could still go in the domain?

Reply:
You mean templates for response selector right?

Reply:
Sorry yes,  meant response selector responses.

Reply:
They're using the same key as the domain responses at any rate

Reply:
No, currently that isn't implemented according to https://github.com/RasaHQ/rasa/issues/6566.

Reply:
Yep, but the responses defined in the domain are not available in the training data object right now.

Reply:
I gotcha. That's fine then docs wise, we're planning on changing it before the release though?

Reply:
@degiz will have a better answer to this :)

Reply:
@dakshvar22 thinking about this: can't we also fix this in the importer, read the domain and add these responses to the nlu training data?

Reply:
We're planing on being able to use the same `responses` key for both "response selector templates" and "domain responses", but that would be post RC1 thing to do. I think this PR should get it first.

Reply:
so atm, the responses could go in e.g. nlu.yml with an `nlu` section and a `responses` section, but not in domain.yml with those `responses` right?

Reply:
@tmbo Yes, I just implemented the logic in importer to keep the response templates always in sync between NLU `TrainingData` and `Domain` objects irrespective of where they are defined(a separate file or domain file).

Reply:
@melindaloubser1 The constraint is no longer true. The responses can go in the `responses` section of any file now.